README.md

```diff
@@ -157,6 +157,7 @@ _See also [Internet of Things (IoT)](#internet-of-things-iot)_
 - [MeTube](https://github.com/alexta69/metube) - Web GUI for youtube-dl, with playlist support. Allows downloading videos from dozens of websites. `AGPL-3.0` `Python/Nodejs/Docker`
 - [n8n](https://n8n.io/) - Free node based Workflow Automation Tool. Easily automate tasks across different services. ([Source Code](https://github.com/n8n-io/n8n)) `Apache-2.0` `Nodejs`
 - [nefarious](https://github.com/lardbit/nefarious) - Web application that automates downloading Movies and TV Shows. `GPL-3.0` `Python`
+- [OliveTin](https://github.com/jamesread/OliveTin) - OliveTin is a web interface for running Linux shell commands. ([Demo](https://www.youtube.com/watch?v=Ej6NM9rmZtk)) `AGPL-3.0` `Go`
```

Comment:
`Demo` links should only be used for interactive demos, i.e. not video demonstrations
 (https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2459). Please remove the demo link.

Reply:
I understand, sorry for the confusion. I didn't see this in the rules but could have checked the other demo examples. Commit b5cbdea should address this.